src/components/Review.jsx

```diff
@@ -49,6 +61,56 @@ const Review = () => {
     });
   };
 
+  const handleCommentSubmit = event => {
```

Comment:
To prevent multiple comments on slow connections, we would need to also disable the form whilst the post request is submitting.